src/static/css/ebook.css

```diff
@@ -465,6 +479,10 @@ section.chapter {
     line-height: none;
   }
 
+  .authors .social .fn::footnote-call {
+    vertical-align: sub;
```

Comment:
Text links use superscript. Why give social icons subscript?

Reply:
It’s because the images are vertically centred according to the text and, as the images were larger than the line-height, the superscript actually looked like it was the middle of the images. Subscript looked cleaner. Did look at changing all links to subscript but didn’t really like it. Could look at reverting this to superscript and then moving it up a bit more so it really is superscript to the images if you want?

Reply:
This is fixed (and screenshot in first comment updated) but still required a manual vertical-height to be set. But at least it's super-script again now so consistent.